mobile/android/android-application-manager.ts

```diff
@@ -49,10 +50,39 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
 			"1"]);
 
 		if (!this.$options.justlaunch) {
+			let processIdentifier = await this.getProcessId(appIdentifier);
+			if (processIdentifier) {
+				let deviceIdentifier = this.adb.identifier;
+				this.$deviceLogProvider.setApplicationPidForDevice(deviceIdentifier, processIdentifier);
+			}
+
 			await this.$logcatHelper.start(this.identifier);
 		}
 	}
 
+	private async getProcessId(appId: string): Promise<string> {
+		// Read the process id of the application that was just started
+		// Information is returned in the following format
+		// USER      PID   PPID  VSIZE  RSS   WCHAN            PC  NAME
+		// u0_a64    4831  488   851380 94888    ep_poll f1e43bb9 S org.nativescript.myApplication
+		let processes = await this.adb.executeShellCommand(["ps"]);
```

Comment:
Instead of executing this and parsing the result, can't you use the `$androidProcessService`: https://github.com/telerik/mobile-cli-lib/blob/master/mobile/mobile-core/android-process-service.ts#L206-L218